### DISCUSSION TITLE
example.com/<slug> for examples/blog-starter-typescript

### DISCUSSION THREAD

**Discussion by sebdanielsson** [vercel/next.js#25050] [Help] [answered]
Hi!

I just tried the examples/blog-starter-typescript which seems really nice but I noticed that all posts are located at example.com/posts/<slug> instead of example.com/<slug> that I use today. For SEO reasons I would the URLs to be the same after the migration. Any tips on how to get this done?

Best regards
Sebastian

**Comment by bobarros** [comment, marked as answer]
Just remove the file [slug].tsx from the directory 'posts'. You can even delete the directory.
Update the paths into the file and it's done.

Folders structure inside the directory 'page' it's going to define your subpaths structure.

Right now the example has [slug].tsx  a posts/dynamic-routing page. By moving the file one directory above, you now have /dynamic-routing

**Reply by sebdanielsson** [reply]
Thanks, fixed it!🎉
